main.go

```diff
@@ -98,7 +98,7 @@ func main() {
 		&upgradeConfig.UpgradeID,
 		"upgrade-id",
 		"",
-		"Unique identifier used to resume a partial upgrade (optional)",
+		"Unique identifier used to resume a partial upgrade (required)",
```

Comment:
You should also be able to use https://godoc.org/github.com/spf13/cobra#MarkFlagRequired

Reply:
We had that originally, but we can't use it because the flags themselves are not required when using `--config`

Reply:
Got it, make sense!